# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1302653235 14400
#      Tue Apr 12 20:07:15 2011 -0400
# Branch stable
# Node ID b1913cd27f76c8c52c576d3f3d7c434569d7629a
# Parent  e12a7c0a1fc51d99fca565070ff4876d761b758f
data.txi: Replace xref{Data Structures} with xref{Structures}.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2011-04-12  Ben Abbott  <bpabbott@mac.com>
+
+	* interpreter/data.txi: Replace xref{Data Structures} with
+	xref{Structures}.
+
 2011-04-11  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/func.txi: Add discussion of isargout to Ignoring
 	Arguments section of documentation.
 
 2011-04-07  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am: Add spellcheck target to documentation
diff --git a/doc/interpreter/data.txi b/doc/interpreter/data.txi
--- a/doc/interpreter/data.txi
+++ b/doc/interpreter/data.txi
@@ -141,17 +141,17 @@ indexing operations that work for matrix
 @cindex structures
 @cindex data structures
 
 Octave's data structure type can help you to organize related objects of
 different types.  The current implementation uses an associative array
 with indices limited to strings, but the syntax is more like C-style
 structures.
 
-@xref{Data Structures}, for more information.
+@xref{Structures}, for more information.
 
 @node Cell Array Objects
 @subsection Cell Array Objects
 @cindex cell arrays
 
 A Cell Array in Octave is general array that can hold any number of
 different data types.
 
